eng/pipelines/aggregate-reports.yml

```diff
@@ -98,16 +98,21 @@ jobs:
       inputs:
         suppressionsFile: 'eng\CredScanSuppression.json'
       condition: succeededOrFailed()
-    - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
-      displayName: 'Publish Security Analysis Logs'
     - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
       displayName: 'Post Analysis'
       inputs:
         GdnBreakAllTools: false
         GdnBreakGdnToolCredScan: true
         GdnBreakGdnToolCredScanSeverity: Error
-        GdnBreakBaselineFiles: $(Build.SourcesDirectory)/eng/java.gdnbaselines
-        GdnBreakBaselines: baseline      
+        GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
+        GdnBreakBaselines: baseline  
+        # Used for generating baseline file.
+        # GdnBreakOutputBaselineFile: java
+        # GdnBreakOutputBaseline: baseline
+      condition: succeededOrFailed()
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+      displayName: 'Publish Security Analysis Logs'
+      continueOnError: true
```

Comment:
Do you want this to continue on error? I thought you baselined things so that shouldn't be needed.

Reply:
Good call. Will remove